### PR TITLE
Add AmountInput and AssetSelectorButton, Keypad and Quick amounts components

### PR DESCRIPTION
**Pull request by wachunei**
**Description**

This PR adds AmountInput, AssetSelectorButton, Keypad and QuickAmounts components to the Storybook

### CODE REVIEW

File: app/components/UI/FiatOnRampAggregator/components/AmountInput.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	},
+	chevron: {
+		flex: 0,
+		marginLeft: 8,
+	},
+});
+
+interface Props {
+	label?: string;
+	currencySymbol: string;
```
Comment (wachunei) [inline app/components/UI/FiatOnRampAggregator/components/AmountInput.tsx:25]: This has to be optional (when currency is crypto)

File: app/components/UI/FiatOnRampAggregator/components/AmountInput.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	highlighted,
+	onPress,
+	onCurrencyPress,
+}: Props) => (
+	<Box label={label} onPress={onPress} highlighted={highlighted}>
+		<View>
+			<ListItem.Content>
+				<ListItem.Body>
+					<Text black bold style={styles.amount} numberOfLines={1} adjustsFontSizeToFit>
+						{currencySymbol}
```
Comment (wachunei) [inline app/components/UI/FiatOnRampAggregator/components/AmountInput.tsx:47]: Optional, fallback to empty string

File: app/components/UI/FiatOnRampAggregator/components/AmountInput.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	label,
+	currencySymbol,
+	amount,
+	currencyCode,
+	highlighted,
+	onPress,
+	onCurrencyPress,
+}: Props) => (
+	<Box label={label} onPress={onPress} highlighted={highlighted}>
+		<View>
```
Comment (wachunei) [inline app/components/UI/FiatOnRampAggregator/components/AmountInput.tsx:43]: Perhaps this view can be removed.

File: app/components/UI/FiatOnRampAggregator/components/Box.stories.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { SafeAreaView } from 'react-native';
+
+import { storiesOf } from '@storybook/react-native';
+
+import { action } from '@storybook/addon-actions';
+import { text, boolean } from '@storybook/addon-knobs';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
```
Comment (alaahd) [inline app/components/UI/FiatOnRampAggregator/components/Box.stories.js:8]: Wouldn't be more optimized to use named import instead of default import ?

**Pull request merged**